server/controllers/profile.js

```diff
@@ -0,0 +1,53 @@
+const Profile = require("../models/ProfileModel");
+
+// create new profile using given arguments
+const addProfile = async (props) => {
+  let newProfile = await new Profile(props);
+  let profile = await newProfile.save();
+  return profile;
+};
```

Comment:
Not sure you need this part. Everything can be done in `createProfile` :smile:

Reply:
I think because the ticket ask for giving parameters so I used a separate function with parameter then passed in request body as argument inside createProfile

Reply:
As for the front-end code, I think this is from the previous commit .
if you can help to exclude files from the previous commit.
thanks

Reply:
Hey @ashrafcoding, you can include the above code as part of the `createProfile` controller. The ticket refers to parameters - which is nothing but request body parameters :)

Reply:
Also, since you've based this branch off of a previous branch - you can adjust the base of this PR to only reflect new changes. 

Ref: https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request